libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -72,6 +73,16 @@ public static class ExpressionFunctions
         /// </summary>
         private static readonly object _randomizerLock = new object();
 
+        /// <summary>
+        /// Culture Info of current thread.
+        /// </summary>
+        private static readonly CultureInfo Locale = CultureInfo.InvariantCulture;
+
+        /// <summary>
+        /// The Path stores the locale info in memory.
+        /// </summary>
+        private static readonly string LocaleMemoryPath = "turn.activity.locale";
```

Comment:
I mean, here, this knowledge "turn.activity.locale" should not flow into expression library.

Reply:
moved to ActionScope